gcp/cloudsql/cloudsql.go

```diff
@@ -31,6 +32,6 @@ var CertSourceSet = wire.NewSet(
 
 // NewCertSource creates a local certificate source that uses the given
 // HTTP client. The client is assumed to make authenticated requests.
-func NewCertSource(c *gcp.HTTPClient) *certs.RemoteCertSource {
-	return certs.NewCertSourceOpts(&c.Client, certs.RemoteOpts{})
+func NewCertSource(c *gcp.HTTPClient, t oauth2.TokenSource) *certs.RemoteCertSource {
```

Comment:
This looks like a non-compatible change. Can you leave this function alone and add a new one instead?  E.g., "NewCertSourceWithIAM" or some such.

Reply:
Yes, that makes sense.